webpack.config.js

```diff
@@ -1,5 +1,6 @@
-
+const path = require('path');
 module.exports = {
+  entry: path.resolve(__dirname, 'src') + '/index.js',
```

Comment:
we should use a client and server folder within the source and point the entry to the client/index.js instead of the index.js in the src folder